### DISCUSSION TITLE
Expire Tags analytics ?

### DISCUSSION THREAD

**Discussion by martinr103** [yeti-platform/yeti#997] [General]
Hey,
I have a question.
We were using Yeti 1.x for a few years now... And I've found it _very_ important to have the Analytics job "Expire tags" up and running properly. Because otherwise, the amount of IOCs is growing and growing... and if you want to export them and use them elsewhere (SIEM/SOC) then of course you want to have the number of IOCs under control.
Now, I am just checking out version 2.x ... and I don't see the "Expire Tags" job in the Analytics pane anymore.
Why is that ?? What happened to "Expire Tags" in v2.x  ???
Cheers!
Martin

**Comment by tomchop** [comment]
Hey @martinr103 ! Yes, we moved ExpireTags to the `deprecated` directory because we just didn't migrate it back to v2. But it seems to be used by people (i.e. you) so we'll prioritize bringing it back shortly (in a few days, maybe sooner). Sorry for the confusion!

**Reply by martinr103** [reply]
Hey @tomchop , Thanks for your reply!
This is good news!  I didn't know that you had some of the migrations still on the to-do list.
And at least in our "use case" - this functionality is really quite important.
So yeah, I am really glad to hear, that you are prioritizing it, and that it should be available again soon! 👍 
Cheers!

**Comment by tomchop** [comment]
PR here: https://github.com/yeti-platform/yeti/pull/1000

**Reply by tomchop** [reply]
Merged! This will be part of this week's release :)

**Reply by martinr103** [reply]
Great stuff!
